src/OrchardCore.Modules/OrchardCore.AdminDashboard/Migrations.cs

```diff
@@ -37,7 +43,20 @@ public async Task<int> UpdateFrom1Async()
         {
             await _recipeMigrator.ExecuteAsync("dashboard-widgets.recipe.json", this);
 
-            return 2;
+            // Shortcut other migration steps on new content definition schemas.
+            return 3;
         }
+
+        // This code can be removed in a later version.
+        public int UpdateFrom2()
+        {
+            SchemaBuilder.AlterIndexTable<DashboardPartIndex>(table => table
+                .CreateIndex($"IDX_{nameof(DashboardPartIndex)}_{nameof(DashboardPartIndex.Position)}",
```

Comment:
An index table has at least one `IDX_SomeIndex_DocumentId` index that may contain all columns, even if it is only one more column as here, so i would have named it `IDX_DashboardPartIndex_DocumentId`, as for layer

            SchemaBuilder.AlterIndexTable<LayerMetadataIndex>(table => table
                .CreateIndex("IDX_LayerMetadataIndex_DocumentId", "DocumentId", "Zone")
            );

Note: As suggested by @Skrypt, maybe in the future we could also reduce the names by removing `IDX_` and / or `Index`

Reply:
I would keep the prefix "IDX_" and remove the "Index" word since it is redundant.

Reply:
Combination of both suggestions I hope?